fix: course selection carousels

nikomakela opened the pull request:
HH-266.
HCRC-lib's event selection carousel did not work properly for the course-event-type.
The event-type must always be given when fetching courses - even then when fetching with direct ids.

terovirtanen commented:
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-100-hh-266-fix-course-selection-carousel.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**Events-Helsinki branch is deployed to platta: https://tapahtumat-100-hh-266-fix-course-selection-carousel.dev.hel.ninja** :rocket::rocket::rocket:

nikomakela committed:
test: fix the missing locationExtraInfo

nikomakela committed:
fix: course selection carousels

HH-266.
HCRC-lib's event selection carousel did not work properly for the course-event-type.
The event-type must always be given when fetching courses - even then when fetching with direct ids.
fix: the translation types

terovirtanen commented:
**Events-Helsinki branch is deployed to platta: https://tapahtumat-100-hh-266-fix-course-selection-carousel.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-100-hh-266-fix-course-selection-carousel.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**Sports-Helsinki branch is deployed to platta: https://liikunta-100-hh-266-fix-course-selection-carousel.dev.hel.ninja** :rocket::rocket::rocket:
